src/test/java/org/kainos/ea/service/JobRoleServiceTest.java

```diff
@@ -59,6 +63,68 @@ void getJobRolesShouldThrowDatabaseConnectionExceptionWhenDaoThrowsDatabaseConne
 
     }
 
+    @Test
```

Comment:
you're not mocking out your validator anywhere in here. need a unit test for what happens when validation fails